config/setup.sh

```diff
@@ -32,12 +33,12 @@ fi
 
 userprimarygroup="$( id -Gn "${trueuser}" | cut -d' ' -f1 )"
 arch="$( uname -m )"
-if [ "${os}" == "\"void\"" ]; then
+if [ "${os}" == "manjaro" ]; then
```

Comment:
This line needs to preserve the void check. It should be if void or if manjaro then